Makefile

```diff
@@ -32,7 +32,7 @@ lint: ## run linter(s)
 	@echo "Linting..."
 	@gometalinter --config=gometalinter.json ./...
 
-test-e2e: bin/$(BIN_NAME) ## run end-to-end tests
+test-e2e: bin/$(BIN_NAME) bin/yamlschema ## run end-to-end tests
```

Comment:
`yamlschema` is built here, I don't think it's needed to do it manually again from the `Jenkinsfile`, is it?

Reply:
Well, as test-e2e is not invoked from the Jenkinsfile... Yes it has to be manually built...